CLI.md

```diff
@@ -322,3 +339,17 @@ permitted.
   entries, if `--factomd` is specified.
 - `--curl` -  Do not submit the Factom entry; print curl commands instead!
 - `--force` - Skip sanity checks for balances, chain status, and sk1 key
+
+**Example Commands**
+
+FAT-0 Coinbase Transaction
+
+Use `--sk1` and `--output` for coinbase transactions; no `--input` is specified as the coinbase input address is always `FA1zT4aFpEvcnPqPCigB3fvGu4Q4mTXY22iiuV69DqE1pNhdF2MC`, the public address that corresponds to a private key of all zeroes. This creates 100 new tokens and sends them to the the `FA2gCm...` address.
+
+```fat-cli transact fat0 --output FA2gCmih3PaSYRVMt1jLkdG4Xpo2koebUpQ6FpRRnqw5FfTSN2vW:100 --sk1 sk1... --ecadr EC3cQ1QnsE5rKWR1B5mzVHdTkAReK5kJwaQn5meXzU9wANyk7Aej```
```

Comment:
You need to break the line after ``` otherwise the rendering is not the same as the other commands above (you can preview the markdown and you will see).

Reply:
Good catch. :-)